db/schema.prisma

```diff
@@ -10,6 +10,14 @@ datasource db {
     url      = env("MYSQL_URL")
 }
 
+model User {
+    id        Int      @id @default(autoincrement())
+    username  String   @unique @db.VarChar(20)
+    password  String
```

Comment:
Missing `@db.VarChar(96)` declaration (from the deleted [feat/db](https://github.com/3x071c/lsg/blob/be603dec32134c2b4055ef0cf9fb0e34ac2a09ea/db/schema.prisma#L11-L15) branch)